components/layout/grid/src/_settings.scss

```diff
@@ -0,0 +1,67 @@
+$lg-breakpoints: $breakpoints !default;
+$lg-column-gap: $p-l !default;
+$lg-columns: 12 !default;
+
+$align-items: center flex-start flex-end;
+
+$justify-content: flex-start center flex-end space-between space-around;
+
+@mixin getFlexBasis($size) {
```

Comment:
in case these mixins are meant to only be used in this component it might make sense to add some kind of scope to the names in order to avoid collitions as there are too generic. In case there are reusable in other components there should be moved to the sui-theme repo

Reply:
Exactly, it's something we have to decide.

We will have to move those mixins to the `sui-theme` so that they can be used without using the component.

We discovered this:
https://github.com/SUI-Components/sui-theme/blob/master/src/layout/_grid.scss

But it looks like it's not being used, so we'll update it with the new mixins changes :)